flutter_local_notifications/lib/src/platform_specifics/darwin/notification_details.dart

```diff
@@ -84,4 +86,10 @@ class DarwinNotificationDetails {
   /// This must refer to a [DarwinNotificationCategory] identifier configured
   /// via [InitializationSettings].
   final String? categoryIdentifier;
+
+  /// The interruption level that indicates the priority and
+  /// delivery timing of a notification.
+  ///
+  /// This property is only applicable to iOS 15.0 or newer.
```

Comment:
As this class is for both iOS and macOS, could you add the macOS version too?

Reply:
Sure, adding comment.